apollo-mockserver/src/appleMain/kotlin/com/apollographql/apollo3/mockserver/MockServer.kt

```diff
@@ -31,10 +31,11 @@ import kotlin.native.concurrent.freeze
  * call. Can be used to simulate slow connections.
  */
 actual class MockServer(
-    private val acceptDelayMillis: Long
-): MockServerInterface {
+    private val acceptDelayMillis: Long,
+    mockDispatcher: MockDispatcher = QueueMockDispatcher(),
+) : BaseMockServer(mockDispatcher) {
 
-  actual constructor(): this(0)
+  actual constructor(mockDispatcher: MockDispatcher) : this(0, mockDispatcher)
```

Comment:
I'm slightly concerned about the `"Dispatcher"` naming here since it's also used extensively in coroutines code. May I suggest `"Handler"` instead, like in [Servlets](https://archive.eclipse.org/jetty/8.0.0.M1/apidocs/org/eclipse/jetty/servlet/ServletHandler.html) and [Ktor](https://ktor.io/docs/requests.html) ?

Reply:
Agreed!

Reply:
Are you ok with `MockServerHandler`?  A bit verbose, but I'm afraid the shorter `MockHandler` would be too broad.

Reply:
That works for me. I can also suggest `HttpHandler`. Your call.